### PR TITLE
Add BLE sample

### PR DESCRIPTION
**Pull request by microbit-sam**
Uses the BLE example from micro:bit V1
https://github.com/lancaster-university/microbit-samples/

### CODE REVIEW

File: source/samples/BLETest.cpp [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    // "enabled": 1,                   // 1 means the Bluetooth stack will be included as standard. 0 means it will not.
+    // "pairing_mode": 1,              // 1 means it's possible to go into pairing mode which will include bringing up the Bluetooth stack whilst in that mode.
+    // "open": 0,                      // 1 means there's no Bluetooth security i.e. no need to pair the micro:bit with other devices that want to communicate with it.
+    // "tx_power": 7,                  // Transmission power of the Bluetooth radio. A value of 0 - 7 with 0 the lowest power and 7 the highest power.
+    // "gatt_table_size": "0x700"      // Amount of memory (in hex bytes) set aside for the Bluetooth GATT table
+    // "nested_heap_proportion": 0.75, // Reducing this can sometimes help make enough memory available for all the Bluetooth services you want. Only experiment with this as a last resort.
+
+    // MicrobitConfig.h in yotta_modules\microbit-dal\inc\core contains MICROBIT_BLE_SECURITY_LEVEL which can be set to SECURITY_MODE_ENCRYPTION_WITH_MITM for passkey authentication when
+    // pairing or SECURITY_MODE_ENCRYPTION_NO_MITM to use Just Works pairing.
+
```
Comment (microbit-mark) [inline source/samples/BLETest.cpp:60]: Is this still relevant for V2?

Comment (microbit-sam) [reply source/samples/BLETest.cpp:60]: Ah no, it needs updating to codal.json

**Review by microbit-mark — APPROVED** [review]
Thanks @microbit-sam LGTM

**Comment by finneyj** [comment]
thanks @microbit-sam!

**Pull request merged**